packages/browser/src/loader.js

```diff
@@ -9,9 +9,10 @@
   _namespace,
   _publicKey,
   _sdkBundleUrl,
-  _config
+  _config,
+  _lazy
 ) {
-  var lazy = true;
+  var lazy = _lazy;
   var forceLoad = false;
 
   for (var i = 0; i < document.scripts.length; i++) {
```

Comment:
Just to be clear, we still want this to be overwritable by the script tag here: https://github.com/getsentry/sentry-javascript/blob/5b660cc350086218a60bc3de988ada679b100fe3/packages/browser/src/loader.js#L20

Or should we guard that to be basically:

```js
var lazy = _lazy;

// if lazy is set to false, we cannot opt in?
if (lazy) {
  // check if option out of lazy
  for (...) {}
} 
```

Just thinking, because I guess it would kind of break in the case of performance/replay if this were to be overwritten to `true` again?

Reply:
> Just to be clear, we still want this to be overwritable by the script tag here: 

Yup, the script tag always supercedes this. Here are the scenarios:

1. lazy = true + data-lazy not set -> lazy == true (default for errors)
2. lazy = false + data-lazy not set -> lazy == false (default for replay + perf)
3. lazy = true + data-lazy === 'no' -> lazy == false
4. lazy = false + data-lazy === 'no -> lazy == false

Reply:
you're right 🤦 - I misunderstood the boolean condition ahhhhh

Let me change this

Reply:
Fixed with f09a96582a89ccdfd960fa78d74c1f651c4ff6ad!